src/UI/UIView.cpp

```diff
@@ -933,17 +952,12 @@ void UIView::init_ui(bool is2D, bool lookingGlass)
     {
       fontsize = 3.0;
     }
-    std::cout << "is2d: " << (is2D ? "true" : "false") << std::endl;
+    
     m_menu_handler = new VRMenuHandler(is2D);
-    if (!m_menu_handler)
-    {
-      std::cout << "m_menu_handler: "
-                << "NULL" << std::endl;
-    }
     VRMenu *menu = m_menu_handler->addNewMenu(std::bind(&UIView::draw_ui_callback, this), 1024, 1024, 1, 1, fontsize);
     menu->setMenuPose(glm::mat4(1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 2, -1, 1));
     m_dir_to_save = m_controller_app.get_directory_path().c_str();
-
+    std::cerr << "init_ui 4: " << std::endl;
```

Comment:
Is this a print for development or for the actual user? Delete if it was just for debugging

Reply:
I removed it.